openrtb/iqgmediarating.go

```diff
@@ -0,0 +1,18 @@
+package openrtb
+
+// IQGMediaRating OpenRTB 2.5 Spec 5.19 IQG Media Ratings
+// Media ratings used in describing content based on the IQG 2.1 categorization.
+// Refer to www.iab.com/guidelines/digital-video-suite for more information.
+type IQGMediaRating int
+
+// IQGMediaRating enums
+const (
+	// All Audiences
+	IQGMediaRatingAll IQGMediaRating = 1
```

Comment:
same issue regarding iota

Reply:
I think it is ok to use itoa

Reply:
I've explain it at https://github.com/Vungle/vungo/pull/77#discussion_r411573066 .
More notes:
In fact, you have found several bugs in files which use itoa, but none in files which use literal numbers.
IMO, itoa is used in cases that we don't care about exactly enum values, but only require them unique. Literal numbers should be used in cases that every enum value are exactly specified and should NOT be changed even if they are unique.